db/migrate/20230725143915_add_confirmation_module_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddConfirmationModuleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :unconfirmed_email, :string
+    add_column :users, :confirmation_sent_at, :datetime
+  end
+end
```

Comment:
- **[OPTIONAL]** This file causes migration errors when migrating the db. This file is not needed. Kindly remove it.